8.0/Dockerfile.rhel9

```diff
@@ -60,6 +60,7 @@ ENV PHP_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/php/ \
     PHP_FPM_CONF_D_PATH=/etc/php-fpm.d \
     PHP_FPM_CONF_FILE=www.conf \
     PHP_FPM_RUN_DIR=/run/php-fpm \
+    PHP_CLEAR_ENV=false \
```

Comment:
We should default to clear_env being `true`, as that is what upstream recommends. The variable should instead be used to override the default setting of `true` to `false`

Reply:
I would also recommend to switch the choices to ON/OFF, similar to existing variables.

Reply:
> I would also recommend switching the choices to ON/OFF, similar to the existing variables.

Thanks for the review, but I don't follow this comment. If and only if PHP_CLEAR_ENV is defined as 'true' than the variable is set.

Reply:
See eg. https://github.com/sclorg/s2i-php-container/blob/master/8.0/README.md?plain=1#L166
The variable is a boolean and has either `ON` or `OFF` value. We should mimic this behavior so that it is the same across all our boolean variables.
Does it make more sense now?

Reply:
Though I guess these variables are using the same values as the configuration files are using :/ Maybe instead make a new section specific to fpm (adn add the new variable there) and use `true` and `false` same as the fpm config is using?